lib/handlers.js

```diff
@@ -2,16 +2,19 @@
 
 var http = require('./http')
   , log = require('fh-bunyan').getLogger('sync-rest-handlers')
-  , VError = require('verror');
+  , VError = require('verror')
+  , urlJoin = require('url-join')
+  , buildRoute = require('./build-route');
 
 var DEFAULT_TIMEOUT = (25 * 1000);
 
 module.exports = function (dataset, opts) {
 
   log.debug(
-    'creating MBaaS sync proxy for dataset "%s" with guid "%s"',
+    'creating MBaaS sync proxy for dataset "%s" with guid "%s" prepend "%s"',
```

Comment:
The reason being, I think prepend doesn't read well and could cause a little confusion without full context